docker-compose.yml

```diff
@@ -4,8 +4,16 @@
 # 84xx - debugger ports (see vsc_debugger.py)
 
 version: "3.8"
+# docker-bake doesn't support volume creation, so we define and label them here
```

Comment:
I don't think this comment is really necessary. `docker buildx bake` is about building images; volumes don't apply to that usecase.